app/uk/gov/hmrc/childcarecalculatorfrontend/utils/TCSchemeInEligibilityMsgBuilder.scala

```diff
@@ -67,20 +67,27 @@ class TCSchemeInEligibilityMsgBuilder {
   }
 
   private def messageForPartnerJourneyOnlyPartnerInWork(answers: UserAnswers)(implicit messages: Messages) = {
-
-    if(answers.partnerWorkHours.getOrElse(BigDecimal(0)) < twentyFoursHours)
-      {
+    val parentBenefits = answers.whichBenefitsYouGet
+    if(answers.partnerWorkHours.getOrElse(BigDecimal(0)) < twentyFoursHours) {
+      if(parentBenefits.isDefined) {
```

Comment:
Change this if else to fold as below

parentBenefits.fold(messages("result.tc.not.eligible.partner.journey.hours.less.than.minimum"))(_ =>
        messages("result.tc.not.eligible.partner.journey.hours.less.than.minimum.parent.receiving.benefits"))